Gemfile

```diff
@@ -5,12 +5,22 @@ git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 # Specify your gem's dependencies in ransack-mongoid.gemspec
 gemspec
 
-gem 'activemodel', '~> 4.0', require: false
 
 if ENV['DB'] =~ /mongoid4/
-  gem 'mongoid', '~> 4.0.0', require: false
+  gem 'activemodel', '~> 4.0', require: false
+  gem 'mongoid', '~> 4.0', require: false
+elsif ENV['DB'] =~ /mongoid5/
+  gem 'activemodel', '~> 4.0', require: false
+  gem 'mongoid', '~> 5.0', require: false
+elsif ENV['DB'] =~ /mongoid6/
+  gem 'activemodel', '~> 6.0', require: false
+  gem 'mongoid', '~> 6.0', require: false
+elsif ENV['DB'] =~ /mongoid7/
+  gem 'activemodel', '~> 6.0', require: false
+  gem 'mongoid', '~> 7.0', require: false
 else
-  gem 'mongoid', '~> 5.0.0', require: false
```

Comment:
This gem should probably declare an official range of supported mongoid versions, and explicitly test those in CI. That sounds like something for a separate PR, though.

Reply:
I tried that but it breaks the older version.